getting_started/macos_x86_64.md

```diff
@@ -7,12 +7,10 @@
 In order to develop in Roc, you need to install the Roc CLI,
 which includes the Roc compiler and some helpful utilities.
 
-1. Download the latest nightly from the assets [here](https://github.com/roc-lang/roc/releases).
-
-1. To prevent "roc can't be opened because Apple can't check it...":
+1. 1. Download the latest roc nightly using the terminal:
```

Comment:
There's an extra `1.` here!
Seems like a mistake, as I checked "view file" and it looks a bit funny.